pkg/scheduler/util/utils_test.go

```diff
@@ -122,55 +124,104 @@ func TestMoreImportantPod(t *testing.T) {
 	}
 }
 
-func TestRemoveNominatedNodeName(t *testing.T) {
+func TestClearNominatedNodeName(t *testing.T) {
+	t.Parallel()
```

Comment:
not need to use `t.Parallel()` in k/k.
Ref: https://github.com/kubernetes/kubernetes/pull/107674#discussion_r817125869

Reply:
I didn't know that, thank you!
And I have fixed those.